cypress/TestCases/FireboltCertification/Advertising.feature

```diff
@@ -24,7 +24,7 @@ Feature: Advertising
       When '3rd party app' registers for the 'advertising onPolicyChanged' event using the 'Firebolt' API
       And '3rd party app' invokes the 'Firebolt' API to 'get advertising policy'
       And 1st party app invokes the 'Firebolt' API to '<API_Key>'
-      Then 'Firebolt' platform responds to '1st party app' with 'null for advertising setSkipRestriction'
+      Then 'Firebolt' platform responds to '1st party app' with '<API_Key>'
```

Comment:
Are these supposed to be "responds FOR" for a default response?

Reply:
updated as :  responds to '1st party app' for '<API_Key>'